src/GitHub.VisualStudio/TeamExplorer/Connect/GitHubConnectSection.cs

```diff
@@ -129,23 +182,70 @@ void UpdateConnection()
         void OnPropertyChange(object sender, System.ComponentModel.PropertyChangedEventArgs e)
```

Comment:
You could probably omit the `System.CompenentModel.` part here since it's already declared in the usings section.